test/WebApi2.1-Swashbuckle5/Startup.cs

```diff
@@ -24,19 +29,31 @@ public Startup(IConfiguration configuration)
         public void ConfigureServices(IServiceCollection services)
         {
             services
-                .AddMvc()
+                .AddMvcCore()
+                .AddApiExplorer()
+                .AddVersionedApiExplorer(options => options.GroupNameFormat = "'v'VVV")
+                .AddAuthorization()
```

Comment:
Can you please not call so many .Add() methods

Reply:
Previously you had the group call AddMvc which internally essentially calls what I put there, but with the ability to customize the version format.  I'll see if its still reproduceable with the group call AddMvc